README.md

```diff
@@ -183,7 +183,7 @@ If you need a feature, you're very welcome to [open an issue](https://github.com
 
 Should be straightforward to implement if needed. Maybe `client-zip` should allow extending by third-party code so those extra fields can be plug-ins instead of built into the library.
 
-The UNIX permissions in external attributes (ignored by many readers, though) are hardcoded to 664, could be made configurable.
+The UNIX permissions in external attributes (ignored by many readers, though) are configurable, set by default to 664 for files, 775 for folders.
```

Comment:
Since this is the roadmap, you should `<del>strike through</del>` this part to indicate that is is completed.

Reply:
Updated the README as requested.